_data/investing.yml

```diff
@@ -135,6 +135,8 @@ websites:
     tfa:
       - sms
       - email
```

Comment:
Thanks for the doc link! If email 2FA only is used as a backup method then it shouldn't be listed here at all.
```suggestion
```